cmd/src/gateway_benchmark.go

```diff
@@ -152,7 +178,30 @@ func benchmarkEndpoint(client *http.Client, url string) time.Duration {
 		fmt.Printf("Error reading response body: %v\n", err)
 		return 0
 	}
+	if resp.StatusCode != http.StatusOK {
+		fmt.Printf("non-200 response: %v\n", resp.Status)
+		return 0
+	}
```

Comment:
Maybe check the body as well? I think it's a great call that you checked for response body == "pong" for WebSockets. I think we should do the same here. I'll adjust the endpoints to respond with "pong"